android/src/main/java/com/doublesymmetry/trackplayer/module/MusicModule.kt

```diff
@@ -368,30 +358,25 @@ class MusicModule(reactContext: ReactApplicationContext) : ReactContextBaseJavaM
 
     @ReactMethod
     fun removeUpcomingTracks(callback: Promise) {
-        if (verifyServiceBoundOrReject(callback)) return
-
         musicService.removeUpcomingTracks()
         callback.resolve(null)
     }
 
     @ReactMethod
     fun skip(index: Int, callback: Promise) {
-        if (verifyServiceBoundOrReject(callback)) return
-
         musicService.skip(index)
         callback.resolve(null)
     }
 
     @ReactMethod
     fun skipToNext(callback: Promise) {
-        if (verifyServiceBoundOrReject(callback)) return
         musicService.skipToNext()
         callback.resolve(null)
     }
 
     @ReactMethod
     fun skipToPrevious(callback: Promise) {
```

Comment:
A lot of these functions are called from the notification buttons, which happens when the service is unbound. We should not have these checks here, or the notification buttons would not work if the app is in recents (and unbound).